dev/tests/api-functional/testsuite/Magento/GraphQl/Customer/ChangeCustomerPasswordTest.php

```diff
@@ -165,7 +165,7 @@ public function testChangePasswordIfNewPasswordIsEmpty()
     /**
      * @magentoApiDataFixture Magento/Customer/_files/customer.php
      * @expectedException \Exception
-     * @expectedExceptionMessage The account is locked.
+     * @expectedExceptionMessage The current customer isn't authorized.
```

Comment:
I am not sure that a message should be changed. It's not obvious.

If my account was locked but I get a message like `The current customer isn't authorized.` then I can spend a lot of time for understanding what is wrong with my token? But the real reason will be another.